examples/c/profile.c

```diff
@@ -80,6 +80,10 @@ static void show_stack_trace(__u64 *stack, int stack_sz, pid_t pid)
 		result = blaze_symbolize_kernel_abs_addrs(symbolizer, &src, (const uintptr_t *)stack, stack_sz);
 	}
 
+  if (result == NULL) {
```

Comment:
can you please double-check spaces vs tabs indentation? It looks off, at least here in the UI

Reply:
Changed.